app/views/teachers/test_settings/_test_results.html.slim

```diff
@@ -0,0 +1,18 @@
+.col-md-10
+  table.table.table-bordered.table-condensed.table-striped#table_with_test_results
+    tr
+      th = t '.user_name'
+      th = t '.class'
+      th = t '.result'
+      th = t '.tries_number'
+      th = t '.show'
+
+      - @test.each do |user, number|
+        tr
+          td = user[0]
```

Comment:
I want show elements from this array https://github.com/zdubzdab/schoolTest/commit/2c83e6a3b0327776279ca6e1d10e90a2a6cd1b48#diff-cb5de511110d9ed76ab74f2beaf641a9R39